scripts/vault_login.py

```diff
@@ -22,7 +22,7 @@ def _vault_command(command: str):
     vcommand = subprocess.run(command, capture_output=True, shell=True)
     if vcommand.returncode != 0:
         verror = str(vcommand.stderr)
-        if 'vault: command not found' in verror:
+        if vcommand.returncode == 127:
```

Comment:
Nice Addition ! 

BTW the error messages are being checked to print the right message. But if we have returncodes for every different failure then we should do that for all the errors here !

Reply:
There is a difference that the "command not found" comes from shell which is usually translated, but other error messages returned by _vault_ which is english only. I also noticed that _vault_ returns code 2 for both implemented catches, so there's no other way to differentiate.
[token revoke](https://github.com/hashicorp/vault/blob/d341378c6353a9e0f776ca024215ea11be8672df/command/token_revoke.go#L170)
[token lookup](https://github.com/hashicorp/vault/blob/d341378c6353a9e0f776ca024215ea11be8672df/command/token_lookup.go#L125)

Reply:
Cool! Lets only have return code check for `command not found`, rest we will check for `error string`. Agree @rmynar ?

Reply:
Agree. Thanks for review.